script/_otbr

```diff
@@ -68,8 +68,8 @@ otbr_install()
         "-DOTBR_DBUS=ON"
         "-DOTBR_DNSSD_DISCOVERY_PROXY=ON"
         "-DOTBR_SRP_ADVERTISING_PROXY=ON"
-        "-DOTBR_MDNS=mDNSResponder"
         "-DOTBR_INFRA_IF_NAME=${INFRA_IF_NAME}"
+        "-DOTBR_MDNS=${OTBR_MDNS:=mDNSResponder}"
```

Comment:
```suggestion
        "-DOTBR_MDNS=${OTBR_MDNS:-mDNSResponder}"
```

Should be `:-` ?

Reply:
I'm inclined to `:=`. If `OTBR_MDNS` is used again later in the script, we don't need to set a default value again.

Reply:
SGTM